chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1299,8 +1306,10 @@ impl PeerManagerActor {
         msg: PeerManagerMessageRequest,
         ctx: &mut Context<Self>,
     ) -> PeerManagerMessageResponse {
+        let msg_type: &str = (&msg).into();
         let _span =
-            tracing::trace_span!(target: "network", "handle_peer_manager_message").entered();
+            tracing::trace_span!(target: "network", "handle_peer_manager_message", msg_type)
+                .entered();
```

Comment:
Hm, there is already a span introduced by 

```
impl Handler<WithSpanContext<PeerManagerMessageRequest>> for PeerManagerActor
```

`handle` method of which almost directly delegates to this method. This makes me wonder if the instrumentation here isn’t being excessive – what value does this span provide that isn’t provided by the parent span? Are there any other callers for this method? Would it make sense to just remove this instrumentation point altogether?

Same questions for other similar occurrences in this file.

Reply:
Good point, removed this span.

> similar occurences

Checked other spans in this file. They all represent sub-cases of processing a certain message, or have multiple callsites.